### PR TITLE
Update helium-network.mdx

### PR DESCRIPTION
**Pull request by waveform06**
updated with a questions shuffle and replacing HNT with IOT and removal of "Helium Blockchain"

### CODE REVIEW

File: docs/faq/helium-network.mdx [hunk 1]
```diff
@@ -13,70 +13,66 @@ slug: /faq/helium-network
...
-uses sub-GHz radio frequency. Depending on conditions, Helium LoRaWAN can reach miles and even
-connect devices that are inside buildings unlike GPS and cellular signals.
+uses sub-GHz radio frequency between 860 and 923 MHz. Depending on conditions, Helium LoRaWAN can 
+reach dozens of miles or kilometers and even connect devices that are inside buildings unlike GPS and 
+cellular signals.
 
 ### What kind of cryptocurrency do I earn?
 
-Hotspot owners earn HNT tokens that are unique to the Helium blockchain.
+Hotspot owners earn IOT tokens on the Solana Blockchain
```
Comment (jthiller) [inline docs/faq/helium-network.mdx]: Lets leave off the mention of Solana here.
We also need to illustrate Mobile Hotspots as entity on the network.

We can link to the IOT and MOBILE pages for more detail.

File: docs/faq/helium-network.mdx [hunk 2]
```diff
@@ -13,70 +13,66 @@ slug: /faq/helium-network
...
 
-### How do Hotspots earn HNT?
+### How do Hotspots earn IOT?
 
-Hotspots earn HNT by providing wireless coverage and transferring data from nearby devices. The more
-data transferred the larger the reward.
-[Here’s a helpful video about how HNT works](https://www.youtube.com/watch?v=B0JOB88MHSI).
+Hotspots earn IO by providing wireless coverage and transferring data from nearby devices. The more
```
Comment (jthiller) [inline docs/faq/helium-network.mdx]: typo here

File: docs/faq/helium-network.mdx [hunk 3]
```diff
@@ -13,70 +13,66 @@ slug: /faq/helium-network
...
+data transferred the larger the reward. Previously hotspots used to earn HNT tokens and
```
Comment (jthiller) [inline docs/faq/helium-network.mdx]:
```suggestion
data transferred the larger the reward. Previously Hotspots earned HNT tokens and
```

File: docs/faq/helium-network.mdx [hunk 4]
```diff
@@ -91,17 +87,21 @@ Hotspot owners earn HNT depending on how frequently their devices do two things:
...
+token economics [here](https://www.helium.com/token).
 
-### Who governs the Helium Network and what is the Helium Foundation?
+### Can I connect my phone or computer to Hotspots?
 
-The Helium Foundation is a global not-for-profit dedicated to the proliferation of secure,
-cost-effective connectivity and it manages and maintains the open technologies that support the
-Helium Network. You can read more about governance from Helium Foundation
-[here](https://helium.foundation).
+Yes. As of early 2022 the community has started to roll out 5G-CBRS hotspots in the USA
```
Comment (jthiller) [inline docs/faq/helium-network.mdx]: Capitalize Hotspot

File: docs/faq/helium-network.mdx [hunk 5]
```diff
@@ -91,17 +87,21 @@ Hotspot owners earn HNT depending on how frequently their devices do two things:
...
+[4G/LTE and 5G coverage on the Helium blockchain](https://www.helium.com/5G).
+
+### Does Helium partner with Internet providers?
+
+Yes. In fact, Helium recently partnered with DISH, a nationwide provider of internet connectivity,
```
Comment (jthiller) [inline docs/faq/helium-network.mdx]: 2021 isn't exactly recent. 
Is this a common question? If we feel like we need to keep it, t-mobile might be more relevant

**Review by samgutentag — APPROVED** [review]

**Pull request merged**